locales/templates/app.pot

```diff
@@ -873,10 +875,9 @@ msgctxt "share_app_dialog_btn_share"
 msgid "Share"
 msgstr ""
 
+#, c-format
 msgctxt "share_app_promotion_text"
-msgid ""
-"Hey, check out this fast and lightweight browser from Mozilla. "
-"{{app_name}} {{share_app_google_play_url}}"
+msgid "Hey, check out this fast and lightweight browser from Mozilla. %1$s %2$s"
```

Comment:
Same comment as The "Rocket" one for "Mozilla" here. Are we sure we want this to be changeable? 

Also, please note in the localization note what %1$s %2$s each stand for

Reply:
Hmm yeah we were missing the Mozilla part in various earlier string generations. I'll try to fix them with a separate update.